artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQServerLogger.java

```diff
@@ -719,7 +719,7 @@ void slowConsumerDetected(String sessionID,
    @Message(id = 222094, value = "Bridge unable to send message {0}, will try again once bridge reconnects", format = Message.Format.MESSAGE_FORMAT)
    void bridgeUnableToSendMessage(@Cause Exception e, MessageReference ref);
 
-   @LogMessage(level = Logger.Level.WARN)
+   @LogMessage(level = Logger.Level.INFO)
```

Comment:
We had a talk on IRC about this:

```
dtikhomi: customer is parsing logs, and in that case warn is false alarm for him
[09:42am] clebert: but the thing is...
[09:42am] clebert: when that issue is actually happening for a bad cause
[09:42am] clebert: he won’t be alarmed for the real issue
[09:42am] clebert: so .. that’s my concern
[09:42am] clebert: nodes stop because of network outages...
[09:43am] clebert: he won’t get notice
[09:43am] dtikhomi: ohhh, i got it
[09:43am] clebert: or missing connectiong because of outages
[09:43am] clebert: u agree with me?
[09:43am] dtikhomi: plz dont close PR i will update it
[09:43am] dtikhomi: yeap, you right
[09:44am] clebert: but do you think you know how to fix it with an easy fix?
[09:44am] clebert: I thought it would be more substantial for a proper fix
[09:44am] clebert: (like… is Shutting Down somewhere… ignore it)
[09:44am] clebert: that’s actually the fix
[09:44am] clebert: server.isStopping
[09:44am] clebert: —> don’t log it
[09:44am] clebert: or log.debug
[09:44am] dtikhomi: yeap, i wonna try
[09:45am] clebert: ok.. thanks a lot...
```